roles/docker/templates/custom.conf.j2

```diff
@@ -3,3 +3,4 @@
 [Unit]
 Wants=iptables.service
 After=iptables.service
+PartOf=iptables.service
```

Comment:
@danwinship I think you said we didn't want this because stopping iptables shouldn't stop docker?

Reply:
Yeah. I think the Wants/After already does what we want in the normal case (like, after rebooting), but that file is getting written after iptables.service and docker.service are both already running, so it doesn't have any effect during the initial install.

Michael had said on IRC that docker was supposed to be getting restarted after that point anyway, when /etc/containers/registries.conf got edited, but that didn't seem to happen on this machine:

    ● docker.service - Docker Application Container Engine
       Active: active (running) since Thu 2017-10-05 17:49:07 UTC; 1h 8min ago

vs

    -rw-r--r--. 1 root root 836 Oct  5 17:50 /etc/containers/registries.conf

Reply:
@michaelgugino Lets drop this change and then LGTM